### DISCUSSION TITLE
Ability to change how many page links are shown with Pagination component

### DISCUSSION THREAD

**Discussion by tsteuwer** [primefaces/community#959] [PrimeNG] [answered]
I think it would be great if we had the ability to set the number of paginator links that can be shown.

By default, it always shows 5.

![image](https://github.com/primefaces/community/assets/32495798/a31f675c-ef24-433b-b7ca-ae07aaf469e5)

There are certain situations where you might only show 3 (e.g. mobile), or others where you could show up to 10. It all depends on how you want to design (or maybe your design team wants to show more). See example:

![image](https://github.com/primefaces/community/assets/6433741/b227f250-187c-4aca-8c6a-49c36093c2a9)

I wouldn't mind implementing this if you take PRs from outside sources.

This would be very useful in the Table component as well, to be able to override showing 5 (and others).

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

You can use `pageLinkSize` property https://primeng.org/paginator#api.paginator.props.pageLinkSize
